README.md

```diff
@@ -5,11 +5,11 @@ contains a runner and language-specific harnesses to confirm feature behavior ac
 
 These SDK features serve several purposes:
 
-* Ensure parity across SDKs by having same-feature snippets adjacent to one another
-* Confirm feature behavior across SDK versions
-* Confirm history across SDK versions
-* Document features in different SDKs
-* Easy-to-use environment for writing quick workflows in all languages/versions
+- Ensure parity across SDKs by having same-feature snippets adjacent to one another
+- Confirm feature behavior across SDK versions
+- Confirm history across SDK versions
+- Document features in different SDKs
+- Easy-to-use environment for writing quick workflows in all languages/versions
```

Comment:
... why lol?

Reply:
I have prettier running on autosave, I didn't even notice it did that